README.md

```diff
@@ -75,17 +75,30 @@ Route::get('/home', 'FooController@bar')->from('ca', 'de', 'fr')->deny();
 
 > ***Note:*** This package uses [*stevebauman*][4]'s [location package][5], please refer to it's [official documentation][6] for a detailed guide on how to configure it correctly.
 
-#### Manual configuration
+### Global Configuration
```

Comment:
Would it be beneficial to move the `php artisan vendor:publish [...]` part here?

Reply:
I think it should be in the installation section where developers can see it before reading the whole docs, what about making a reference here for the "Publish the configuration" point?

Reply:
I have pushed a new commit, please take a second look and tell me what do you think.